common/lib/xmodule/xmodule/js/src/video/00_sjson.js

```diff
@@ -51,11 +67,38 @@ function() {
             return min;
         };
 
+        var filter = function (start, end) {
+            var filteredTimes = [],
+                filteredCaptions = [],
```

Comment:
For cleaner diffs, I usually prefer to make each declaration its own `var` statement. This cuts down on the class of bugs that stem from forgetting to adjust commas and/or semicolons as the list is updated.